qubes-artwork.spec.in

```diff
@@ -43,7 +41,11 @@ BuildRequires:	pycairo
 # see backgrounds/Makefile
 #BuildRequires:	inkscape >= 0.91
 
+Source0: %{name}-%{version}.tar.gz
+
+%if 0%{?qubes_builder}
 %define _builddir %(pwd)
+%endif
```

Comment:
I think this is unneeded even in legacy build (when forced somehow), because there is proper Source0 + `%setup -q`. But it shouldn't hurt, will be just weird.